flycheck.el

```diff
@@ -7759,14 +7769,14 @@ to be checked.  It also sets `package-user-dir' according to
   "Option VALUE filter for `flycheck-emacs-lisp-initialize-packages'."
   (let ((shall-initialize
          (if (eq value 'auto)
-             (or (flycheck-in-user-emacs-directory-p (buffer-file-name))
+             (or (flycheck-in-user-emacs-directory-p
+                  (or buffer-file-name default-directory))
```

Comment:
Are we guaranteed that `default-directory` of the temporary Org source buffer the same as the parent Org buffer?

Reply:
Yup, the manual says "When you create a new buffer that
is not visiting a file, via a command like ‘C-x b’, its default
directory is usually copied from the buffer that was current at the time
(*note Select Buffer::)."

Reply:
Thanks for enlightening me :+1: